backend/src/services/tag.service.ts

```diff
@@ -4,30 +4,32 @@ import TagRepository from '../data/repositories/tag.repository';
 import { HttpError } from '../common/errors/http-error';
 import { HttpCode } from '../common/enums/http-code';
 import { HttpErrorMessage } from '../common/enums/http-error-message';
+import { mapTagToITag } from '../common/mappers/tag/map-tag-to-itag';
 
 export const getAllByWorkspaceId = async (
   workspaceId: string,
 ): Promise<ITag[]> => {
-  return getCustomRepository(TagRepository).findAllByWorkspaceId(workspaceId);
+  const tags = await getCustomRepository(TagRepository).findAllByWorkspaceId(workspaceId);
+  return tags.map(mapTagToITag);
 };
 
-export const create = async (newTag: ITagCreation): Promise<ITag> => {
+export const create = async (workspaceId: string, newTag: ITagCreation): Promise<ITag> => {
```

Comment:
Will it be more neat and easy to read? All information in place, no need to search for ITagCreation description.
```suggestion
export const create = async (workspaceId: string, name: string): Promise<ITag> => {
```